src/topology/algebra/group.lean

```diff
@@ -652,3 +652,10 @@ instance additive.topological_add_group {G} [h : topological_space G]
 instance multiplicative.topological_group {G} [h : topological_space G]
   [add_group G] [topological_add_group G] : @topological_group (multiplicative G) h _ :=
 { continuous_inv := @continuous_neg G _ _ _ }
+
+instance Pi.topological_add_group {C : β → Type*}
+  [∀ b, topological_space (C b)] [∀ b, add_group (C b)] [∀ b, topological_add_group (C b)] :
+  topological_add_group (Π b, C b) :=
+{ continuous_add := continuous_pi (λ i, continuous.add
+    ((continuous_apply i).comp continuous_fst) ((continuous_apply i).comp continuous_snd)),
+  continuous_neg := continuous_pi (λ i, continuous.neg (continuous_apply i)) }
```

Comment:
Could you please add `@[to_additive] pi.has_continuous_mul` (if it's not there yet), then reuse `continuous_add` from that instance (e.g., using `.. pi.has_continuous_mul`)?

Reply:
Otherwise LGTM